ambry-account/src/main/java/com/github/ambry/account/MySqlAccountService.java

```diff
@@ -326,6 +328,8 @@ public void updateAccounts(Collection<Account> accounts) throws AccountServiceEx
     }
     // Tell the world
     publishChangeNotice();
+    containerUpdateConsumers.forEach(containerUpdateConsumer -> containerUpdateConsumer.accept(
```

Comment:
These calls should be made in HelixAccountService too.  Consider adding `AbstractAccountService.notifyAccountUpdateConsumers` and calling from both impls.